### PR TITLE
[DOCS]: Weeks 6, 7 and 8 Blog posts

### PR DESCRIPTION
**Pull request by WassCodeur**
Weeks 6, 7 and 8 Blog posts

### CODE REVIEW

File: docs/source/posts/2024/2024-08-06-week6-wachiou-bouraima.rst [hunk 1]
```diff
@@ -0,0 +1,92 @@
+WEEK 6: Code reviews, relining and crush challenges
+===================================================
+
+.. post:: August 6, 2024
+   :author: Wachiou BOURAIMA
+   :tags: google
+   :category: gsoc
+
+Hello everyone,
+As my Google Summer of Code (GSoC) 2024 journey progresses, week6 has brought me a series of technical challenges and accomplishments. My main focus has been on code reviews, rebasing and commits squasing, with a few notable lessons learned along the way.
```
Comment (itellaetxe) [inline docs/source/posts/2024/2024-08-06-week6-wachiou-bouraima.rst:10]: `squasing`  ->  `squashing`

File: docs/source/posts/2024/2024-08-06-week6-wachiou-bouraima.rst [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+- I performed rebasing and squashing to integrate the latest changes and consolidate commits. This process was challenging due to several conflicts that arose. Resolving these conflicts required a deep dive into Git’s functionality, including:
+
+  - **Conflict Resolution:** Manually resolving merge conflicts that affected several files.
+
+  - **Understanding Git Operations:** Gained hands-on experience with rebasing and squashing, which improved my grasp of version control workflows.
+
+  - **Commit Consolidation:** multiple commits into a single commit to streamline the commit history and enhance readability.
+
+.. code-block:: bash
```
Comment (itellaetxe) [inline docs/source/posts/2024/2024-08-06-week6-wachiou-bouraima.rst:30]: Perhaps you could mention why you introduce this codeblock

File: docs/source/posts/2024/2024-08-06-week6-wachiou-bouraima.rst [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+Acknowledgements
+----------------
+
+I want to extend my thanks to my mentor `Serge Koudoro <https://github.com/skoudoro/>`_, for his detailed feedback and guidance. Your support has been crucial in refining my work. I also appreciate the constructive comments from my peers: `Iñigo Tellaetxe Elorriaga <https://github.com/itellaetxe>`_, `Robin Roy <https://github.com/robinroy03>`_, `Kaustav Deka <https://github.com/deka27>`_, which have been instrumental in improving the quality of my contributions.
+
+
+Did I get stuck anywhere?
+--------------------------
+
+While the rebasing and squashing process presented challenges, I was able to overcome them with the help of My mentor and online resources and documentation.
```
Comment (itellaetxe) [inline docs/source/posts/2024/2024-08-06-week6-wachiou-bouraima.rst:76]: `My` --> `my`

**Review by itellaetxe — COMMENTED** [review]
Thanks for the blogs Wachiou, good work!

Other than some small typos and a comment on a codeblock, LGTM.

File: docs/source/posts/2024/2024-08-06-week6-wachiou-bouraima.rst (hunk 1 above)
Comment (WassCodeur) [reply docs/source/posts/2024/2024-08-06-week6-wachiou-bouraima.rst:10]: Hi @itellaetxe !

Thank you for your comments

File: docs/source/posts/2024/2024-08-06-week6-wachiou-bouraima.rst (hunk 2 above)
Comment (WassCodeur) [reply docs/source/posts/2024/2024-08-06-week6-wachiou-bouraima.rst:30]: By the way, I just wanted to review the command I had to use. I think maybe I should remove it. What do you think ?

File: fury/animation/animation.py [hunk 4]
```diff
@@ -466,7 +466,7 @@ def set_position_interpolator(self, interpolator, *, is_evaluator=False, **kwarg
         --------
         >>> Animation.set_position_interpolator(spline_interpolator, degree=5) # doctest: +SKIP
 
-        """
+        """  # noqa: E501
```
Comment (robinroy03) [inline fury/animation/animation.py:469]: This will go under a different PR. Or update the title a bit to accommodate this. I guess updating the title is enough for this.

**Review by robinroy03 — APPROVED** [review]

**Review by deka27 — COMMENTED** [review]
Yea, as Robin said. The animation.py change should not be in the blog PR. Otherwise, LGTM.

**Comment by WassCodeur** [comment]
Hi @robinroy03, @deka27 !

Thank you for your comments, I'll take care of it right away.

File: fury/animation/animation.py (hunk 4 above)
Comment (WassCodeur) [reply fury/animation/animation.py:469]: Thank you, I think I'll open a pr for it.

**Comment by skoudoro** [comment]
Thank you for this @WassCodeur
